build.gradle

```diff
@@ -441,6 +441,7 @@ project('shared:controller-api') {
 project('controller') {
     sourceSets {
         main.resources.srcDirs += "$projectDir/src/conf"
+        test.resources.srcDirs += "$rootDir/config"
```

Comment:
I think we should decide on the purpose of the `config` directory, because it is apparently playing quite a few roles here.    There's a variety of scenarios:
1. the default configuration file included in the Pravega distribution, for cluster mode.
2. the default configuration file for the standalone mode.
3. the configuration file (and associated data files) for integration tests.

The change seen above (line 444) is obviously related to (3) but it would probably be best to use a proper test resource directory.  Meanwhile, it is questionable that we provide a jks or passwd file.

Reply:
This is correct. About 1 and 2 cluster mode and standalone mode is just two different ways of distributing the same application. They just refer different files in the same `config` directory. Do you recommend finding different locations for configurations of different components?

> Meanwhile, it is questionable that we provide a jks or passwd file.

We have discussed this earlier. This is for user convenience. Admin can create and deploy his own version once he tries the default ones out.

P.S. Both the discussions are not strictly related to the context of the PR :). If we agree this is the way to go, I will log issues for these.

Reply:
Created issue https://github.com/pravega/pravega/issues/2984 to track this as this is not only limited to this test but is applicable for all the integration/unit tests.